kenware/Assessment
# #162187176 create answer endpoints

#### What does this PR do?
- create Answer endpoints

#### Description
- create  POST endpoint
- create GET endpoints
- create PATCH endpoints
- create DELETE endpoint
- add test
#### How should this be manually tested?
- Create a local copy `git ft-create-answer-endpoits-162784678`
- Checkout to the branch `git checkout ft-create-answer-endpoits-162784678`
- Start the application with the command: `python manage.py runserver`
- On postman make a GET or POST request to `{{base-url}}/api/v1/answers/<answer_id>/`
- On postman make a PATCH or DELETE request to `{{base-url}}/api/v1/answers/<answer_id>/`
- An authenticated user should be able to carry out all crude operation on user endpoints
- To run tests use `python manage.py`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
